.helm/ecamp3/templates/api_configmap.yaml

```diff
@@ -6,9 +6,8 @@ metadata:
     {{- include "api.selectorLabels" . | nindent 4 }}
     {{- include "app.commonLabels" . | nindent 4 }}
 data:
-  API_DOMAIN: {{ .Values.api.domain | quote }}
+  API_DOMAIN: {{ .Values.domain | quote }}
```

Comment:
This env variable may be changed after https://github.com/ecamp/ecamp3/pull/3222#discussion_r1111209996

Reply:
done